build.yml

```diff
@@ -111,7 +114,8 @@ jobs:
 
       - bash: |
           set -ex
-          conda build $VERBOSE_OPTION \
+          source activate base
+          mamba build $VERBOSE_OPTION \
```

Comment:
I think the way is to install `boa` and then you run `conda mambabuild` , according to their docs anyway

Reply:
I'm interested if we see any speed up over `mamba build` given the statement from boa docs ```this is equivalent to running conda build my_recipe_folder, but using mamba as a solver```